test/trans.render.icu.spec.js

```diff
@@ -8,19 +8,14 @@ describe('trans using no children but props - icu case', () => {
     <Trans
       defaults="hello <0>{{what}}</0>"
       values={{ what: 'world' }}
-      components={[<strong>univers</strong>]}
+      components={[<strong>universe</strong>]}
     />
   );
   it('should render translated string', () => {
-    const wrapper = mount(<TestElement />);
-    // console.log(wrapper.debug());
-    expect(
-      wrapper.contains(
-        <div>
-          hello <strong>world</strong>
-        </div>,
-      ),
-    ).toBe(true);
+    const { container } = render(<TestElement />);
```

Comment:
technically, `TestElement` variable is a React Component, not an Element.. do we want to rename it together with the migration or ignore for now?

Reply:
Personally, I think `TestComponent` would make much more sense. I'd side with renaming it now.

Reply:
I am also down to rename :sweat_smile: